sdk/tests/core/test_az_json.c

```diff
@@ -1531,6 +1531,18 @@ static void test_json_reader_invalid(void** state)
       AZ_SPAN_FROM_STR("{\r\n\"isActive\":false \"\r\n}"), AZ_ERROR_UNEXPECTED_CHAR);
 }
 
+static void temp_fuzz_test(az_span json, az_result expected_result)
```

Comment:
Just using this to test an example fuzz-tested JSON input in CI, will remove shortly.